pom.xml

```diff
@@ -514,7 +520,7 @@
         <maven.findbugsplugin.version>3.0.5</maven.findbugsplugin.version>
         <identity.workflow.impl.bps.version>5.2.0</identity.workflow.impl.bps.version>
         <maven.checkstyleplugin.excludes>**/gen/**/*</maven.checkstyleplugin.excludes>
-        <identity.event.handler.version>1.3.14</identity.event.handler.version>
+        <identity.event.handler.version>1.3.24-SNAPSHOT</identity.event.handler.version>
```

Comment:
Need to update once https://github.com/wso2-extensions/identity-event-handler-notification/pull/159 is merged and released.